### PR TITLE
Updated readme.me to include information about glob-patterns

### PR DESCRIPTION
**Pull request by themoch**
Updated read me examples to be more clear to developers

Fixes this issue:
#210

### CODE REVIEW

**Comment by RyanZim** [comment]
Should update CLI `--help` output to match.

**Comment by themoch** [comment]
@RyanZim done and done

File: README.md [hunk 1]
```diff
@@ -60,16 +60,16 @@ Options:
...
 
 If no input files are passed, it reads from stdin. If neither -o, --dir, or
 --replace is passed, it writes to stdout.
 
 If there are multiple input files, the --dir or --replace option must be passed.
 
-Input files may contain globs. If you pass an input directory, it will process
-all files in the directory and any subdirectories.
+Input files may contain globs (ie: src/**/*.css). If you pass an input directory, it will process
+all files in the directory and any subdirectories, respecting the glob pattern.
```
Comment (RyanZim) [inline README.md]: Please double-check how the CLI actually outputs this, I'm pretty sure it wraps the lines ~80 chars.

Comment (themoch) [reply README.md]: ah, yes I was unaware that it was defaulted to 80 chars.  I assumed that was just my screen.

Comment (themoch) [reply README.md]: screen shot 2018-04-27 at 9 04 52 am

Does this look ok? if so, window is set to 80 columns.

Comment (themoch) [reply README.md]: @RyanZim  ☝️ that is how it wraps at 80 columns/chars

Comment (RyanZim) [reply README.md]: LGTM

**Comment by RyanZim** [comment]
Sorry, somehow forgot to merge this.

**Pull request merged**